app/src/main/java/org/greenstand/android/TreeTracker/dashboard/DashboardViewModel.kt

```diff
@@ -36,6 +36,7 @@ data class DashboardState(
     val totalTreesToSync: Int = 0,
     val isOrgButtonEnabled: Boolean = false,
     val showUnreadMessageNotification: Boolean = false,
+    val totalTreesToSyncThreshold:Boolean = false
```

Comment:
This should be renamed to something more descriptive like "showTreeSyncReminderDialog"

Reply:
fixed